litert/vendors/intel_openvino/compiler/graph_iterator.h

```diff
@@ -0,0 +1,159 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <unordered_set>
+
+#include <openvino/frontend/tensorflow_lite/decoder.hpp>
+#include <openvino/frontend/tensorflow_lite/graph_iterator.hpp>
+#include <openvino/frontend/tensorflow_lite/quantization_info.hpp>
+#include "litert/c/litert_logging.h"
+#include "litert/c/litert_model.h"
+#include "litert/cc/litert_model.h"
+#include "litert/core/model/model.h"
+
+#include "decoder.h"
+
+using namespace litert;
```

Comment:
please note google style guide prohibits using namespace statements. Either specify for each symbol, or add litert:: prefix